.github/workflows/release_pipeline.yml

```diff
@@ -0,0 +1,72 @@
+name: Release Add-On Pipeline
+
+on: 
+  push:
+    branches: [ "pipeline-integration-test" ]
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  build:
+    runs-on: ${{matrix.os-type}}
+
+    strategy:
+      matrix:
+        os-type: [ windows-2019, macos-latest ]
```

Comment:
Why it is the latest for macos and a specific for windows? Maybe it would be a good idea to use a specific version for macos as well to prevent unexpected issues when the latest image changes.

Reply:
We came to the same conclusion. We will get a specific macos version as well.

Reply:
Fixed MacOS version to latest stable (macos-12 as of today).